statusline/git.py

```diff
@@ -27,10 +27,12 @@ def _run_command(self, command: list) -> Optional[str]:
         except CalledProcessError as e:
             return None
 
-    def _count(self, command: list) -> int:
-        results = self._run_command(command).split("\n")
-        results.remove("")
-        return len(results)
+    def _count(self, command: list) -> Optional[int]:
+        results = self._run_command(command)
+        if results:
```

Comment:
This solution is ugly and not pythonic but has minimal duplication. We have chosen to commit it as an urgent bugfix.

Perhaps we can revisit this solution in future when there is less time pressure.